pyproject.toml

```diff
@@ -70,13 +72,21 @@ optional = true
 
 [tool.poetry.group.docs.dependencies]
 # The latest Sphinx requires a recent Python version.
-Sphinx = { version = ">=8.2", python = "^3.11" }
+Sphinx = [
+  { version = ">=7.4", python = ">=3.9,<3.10" },
+  { version = ">=8.1", python = ">=3.10,<3.11" },
+  { version = ">=8.2", python = "^3.11" },
+]
```

Comment:
The reason I avoided specifying older Sphinx versions was because the older versions don't support %Y for the copyright, but I guess it's okay as long as ReadTheDocs uses the newer Python+Sphinx versions.